dosemu2.spec.in

```diff
@@ -37,6 +37,7 @@ BuildRequires: git >= 2.0
 BuildRequires: xorg-x11-font-utils
 BuildRequires: linuxdoc-tools
 BuildRequires: coreutils
+BuildRequires: readline-devel
```

Comment:
I believe you also need to add `readline`
to `Recommends`.

Reply:
Yes, that's right, unless you can move the
debugger backend code into a shared lib,
as our plugins usually do.
So for now - yes, it should be in `Requires`.